lib/validation/rules/hms.ts

```diff
@@ -1,14 +1,18 @@
 import { helpers } from '@vuelidate/validators'
 import { type Hms } from '../../support/Day'
+import { createRule } from '../Rule'
 import { hms as baseHms } from '../validators/hms'
 
 type HmsType = 'h' | 'm' | 's'
 
+export const message = {
+  en: 'The time is invalid.',
+  ja: '時間表記が正しくありません。'
+}
+
 export function hms(required?: HmsType[], msg?: string) {
-  return helpers.withMessage(
-    () => msg ?? 'The time is invalid.',
-    (value: Hms) => {
-      return !helpers.req(value) || baseHms(value, required)
-    }
-  )
+  return createRule({
+    message: ({ lang }) => msg ?? message[lang],
+    validation: (value: Hms) => !helpers.req(value) || baseHms(value, required)
```

Comment:
`!helpers.req(value)` may not be needed according to `createRule` implementation, but not so sure.

Reply:
Fixed: 5c20e70